entity-services-examples/example-versions/README.md

```diff
@@ -137,24 +137,386 @@ hub-next.xqy is the version-next hub after all clients and sources have been upg
 Running Transitional Hubs
 -------------------------
 
-The process of migrating data forward is complicated somewhat by the fact that a database only has one
-state at a time.  Deployment of index configurations, schemas, and extraction templates cannot be versioned,
-they must be managed as part of the evolution of a hub.
+The process of migrating data forward is complicated somewhat by the fact that
+a database only has one state at a time.  Deployment of index configurations,
+schemas, and extraction templates cannot be versioned, they must be managed as
+part of the evolution of a hub.
 
 This hub scenario handles extraction templates and indexes in different ways.
 
-You may have noticed that example changes data simply by adding a property to the Person type.  It also makes a change
-by adding a word lexicon on that property.  An index change is a change in database state, and so it is either done or not
-done.  Once the index is configured and deployed, applications can use it.  If a model were to change such
-that indexes are no longer needed, the system must maintain both indexes until the data migration is complete.
-In this scenario, we've chosen simply to include the "Model-next" index configuration in the deployment.  Its
-presence does not affect the original hub, and so during the course of this exercise it is latent, only used
-by the 'next' version of search options.
+You may have noticed that example changes data simply by adding a property to
```

Comment:
Are you referring to example-person here? If so it will be nice to call it out to avoid confusion.